Scripts/Python/nb01RPSGame.py

```diff
@@ -710,6 +727,11 @@ def _SendLocalStatusMsg(self, msg):
         """Sends a status chat message (purple text) to the local player's KI."""
         PtSendKIMessage(kKILocalChatStatusMsg, msg)
 
+    def _QuitGame(self, YesNo):
+        if YesNo:
+            PtDisableControlKeyEvents(self.key)
+            PtEnableMovementKeys()
+            PtAvatarExitAFK()
```

Comment:
A comment explaining what this does would be nice (we're not AFK, after all).

Reply:
PTAvatarExitAFK() actually just does a gotostage 2 which works for the sit brain in this case

https://github.com/H-uru/Plasma/blob/9956967363f383d91f43162d116ad3477148adb2/Sources/Plasma/FeatureLib/pfPython/cyAvatar.cpp#L1970

Reply:
Put that in a comment in the code 😉